### PR TITLE
try unflake CI

### PR DESCRIPTION
**Pull request by pmeier**
I went through the commits to `main` the last few days and upped tolerances on flaky tests.

### CODE REVIEW

File: test/test_transforms_v2_consistency.py [hunk 1]
```diff
@@ -354,24 +354,34 @@ def __init__(
...
+        for dt, ckw, extra_args_kwargs in [
+            (torch.uint8, {"rtol": 1e-1, "atol": 1}, []),
+            (
+                torch.float32,
+                {"rtol": 1e-2, "atol": 1e-3},
+                [
+                    # These proved to be flaky on uint8 inputs so we only run them on float32
+                    ArgsKwargs(),
+                    ArgsKwargs(interpolation=v2_transforms.InterpolationMode.BICUBIC),
+                    ArgsKwargs(interpolation=PIL.Image.BICUBIC),
```
Comment (pmeier) [inline test/test_transforms_v2_consistency.py:381]: It seems that with default `alpha` and `sigma`, bilinear (default) and bicubic interpolation are flaky for uint8. For example

- https://github.com/pytorch/vision/actions/runs/4992664184
- https://github.com/pytorch/vision/actions/runs/5043966208

The errors are likely related to some uint8 issue since we have a difference of 255:

```
Tensor-likes are not close!

Mismatched elements: 1 / 318828 (0.0%)
Greatest absolute difference: 255 at index (2, 1, 81, 93) (up to 1 allowed)
Greatest relative difference: inf at index (2, 1, 81, 93) (up to 0.1 allowed)
```

This is not an issue with the testing function though:

```pycon
>>> zeros = torch.zeros(3, dtype=torch.uint8)
>>> ones = torch.ones_like(zeros)
>>> ones - zeros
tensor([1, 1, 1], dtype=torch.uint8)
>>> torch.testing.assert_close(ones, zeros, atol=1, rtol=0)
>>> zeros - ones
tensor([255, 255, 255], dtype=torch.uint8)
>>> torch.testing.assert_close(zeros, ones, atol=1, rtol=0)
```

File: test/test_transforms_v2_consistency.py [hunk 2]
```diff
@@ -354,24 +354,34 @@ def __init__(
...
             ],
             # ElasticTransform needs larger images to avoid the needed internal padding being larger than the actual image
             make_images_kwargs=dict(DEFAULT_MAKE_IMAGES_KWARGS, sizes=[(163, 163), (72, 333), (313, 95)], dtypes=[dt]),
             # We updated gaussian blur kernel generation with a faster and numerically more stable version
             # This brings float32 accumulation visible in elastic transform -> we need to relax consistency tolerance
             closeness_kwargs=ckw,
         )
-        for dt, ckw in [(torch.uint8, {"rtol": 1e-1, "atol": 1}), (torch.float32, {"rtol": 1e-2, "atol": 1e-3})]
+        for dt, ckw, extra_args_kwargs in [
+            (torch.uint8, {"rtol": 1e-1, "atol": 1}, []),
```
Comment (NicolasHug) [inline test/test_transforms_v2_consistency.py:373]: why set rtol to 1e-1 and not 0?

File: test/test_transforms_v2_consistency.py (hunk 1 above)
Comment (pmeier) [reply test/test_transforms_v2_consistency.py:381]: > Is that with bicubic or bilinear? float or ints?

Yes, it is hard and no I'm not happy with it. Please remember that this never was meant to stay longer than the initial release.

Here is the explanation:

The `080` from `ElasticTransform-080` gives us these details:

- `08` (first two digits) it is parametrization with index 8, i.e. the ninth one. Counting this gives us bicubic
  https://github.com/pytorch/vision/blob/4a51822ca20027b6e03ec4fb582c31cc9545ba4e/test/test_transforms_v2_consistency.py#L357-L365
  Similarly, `06` is also bicubic. `00` is bilinear, since that is the default.
- `0` (last digit) is the index `pytest` automatically assigns to avoid duplicate test names. It needs to do that since we have hack in place to instantiate uint8 and float32 test separately but with the same ids:
  https://github.com/pytorch/vision/blob/4a51822ca20027b6e03ec4fb582c31cc9545ba4e/test/test_transforms_v2_consistency.py#L374
  Thus, index `0` corresponds to uint8

> WDYM flaky?

I probably don't understand. No snark answer: error pops up from time to time, but not consistently. Could you rephrase?

> For bicubic we didn't change anythign from v1 to v2. If there are 0-255 differences (overflows?) from v1 to v2 with bilinear that's potentially concerning

They are not new and have been popping up since at least late last year. This is why we have a comment like 

https://github.com/pytorch/vision/blob/4a51822ca20027b6e03ec4fb582c31cc9545ba4e/test/test_transforms_v2_consistency.py#L370-L371

Since we are talking about a single pixel, I don't think that is much of a concern.

File: test/test_transforms_v2_consistency.py (hunk 2 above)
Comment (pmeier) [reply test/test_transforms_v2_consistency.py:373]: Just copied it over from what we had. Came from #6762 that @vfdev-5 authored. Maybe he remembers? Agree that `0` would be better here.

File: test/test_transforms_v2_consistency.py (hunk 1 above)
Comment (NicolasHug) [reply test/test_transforms_v2_consistency.py:381]: Basically I find it concerning that v1 differs significantly from v2 on certain interpolation modes. Those modes are controlled by torch core, not by a v1 vs v2 implementation difference (except for bilinear mode which we changed recently, but according to your comment those problems are pre-dating that change)
